package.json

```diff
@@ -59,6 +59,7 @@
     "react-split-pane": "0.1.89",
     "react-table": "6.10.3",
     "react-tabs": "3.0.0",
+    "react-tooltip-lite": "1.11.1",
```

Comment:
Do we need this library? reactstrap has a Tooltip component (with the same style as in your screenshot) and there's a generic wrapper around it in `ui/shared/SimpleTooltip`.

Reply:
Hmm, good call.  ``SimpleTooltip`` is what worked best here.  I had gotten frustrated with the default ``reactstrap`` tooltip in getting it to layout the content the way I liked and that ``toolsip-lite`` worked quite easily.  

That being said, the ``SimpleTooltip`` works great!  My bad for not checking in the ``/shared`` folder.  I had to add ``autohide`` support to it.  But otherwise, it was perfect.  Thanks.